apps/aecontract/src/aect_create_tx.erl

```diff
@@ -72,6 +77,15 @@
 
 -export_type([tx/0]).
 
+%%%===================================================================
+%%% Conversion of old db format
+
+-spec from_db_format(tx()) -> tx().
+from_db_format(#contract_create_tx{ct_version = #{}} = Tx) ->
+    Tx;
+from_db_format(#contract_create_tx{ct_version = VMVersion} = Tx) ->
```

Comment:
Guard on type of VMVersion would have made this more readable though minor and debatable. Ok.